src/integTest/groovy/com/github/jrubygradle/JRubyPrepareGemsIntegrationSpec.groovy

```diff
@@ -45,6 +47,28 @@ class JRubyPrepareGemsIntegrationSpec extends Specification {
             new File(jrpg.outputDir,"gems/slim-${SLIM_VERSION}").exists()
             new File(jrpg.outputDir,"specifications/.jrubydir").exists()
     }
+  
+    @Ignore // this test does not work with flatRepo since gems needs
```

Comment:
all test which go online to the torquebox proxy are commented out. do not know why !

but getting away from the flatRepo to use the torquebox proxy is a small change